plugins/mqtt/command/read_write.c

```diff
@@ -62,11 +63,14 @@ static int wrap_read_response_json_object(neu_fixed_array_t *   array,
         array->length, sizeof(neu_json_read_resp_tag_t));
 
     for (uint32_t i = 0; i < array->length; i++) {
-        int_val          = neu_fixed_array_get(array, i);
-        val              = int_val->val;
+        // int_val          = neu_fixed_array_get(array, i);
+        // val              = int_val->val;
+        string_val       = neu_fixed_array_get(array, i);
```

Comment:
The original `int_val` is the tag id, should look up the tag name from the datatag table.